server/pages/index.jade

```diff
@@ -236,6 +236,8 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 		script(src=urls[ jsFile ])
 		if chunk
 			script(src=urls[ chunk ])
+		if momentLocaleFile
+			script(src=urls[ momentLocaleFile ])
```

Comment:
@stephanethomas I'm not sure that this code is needed, at least it works correctly without it in the test flow I specified in the PR description.

I think that the original purpose of that is to have the moment-locale data loaded when changing languages as discussed [here](https://github.com/Automattic/wp-calypso/pull/12491#issuecomment-304229400) but I don't entirely see the point because when you change the language the page is reloaded, so `i18n-moment` takes care of the loading again.

Wdyt?

Reply:
I see, thanks for the explanation!